### PR TITLE
Enable wsl1 testing using npm pipeline

### PR DESCRIPTION
**Pull request by ssbarnea**
This is creating a new pipeline because i found the ci.yml very hard to update due to its 1000 lines of code but if someone can take these changes and apply them to ci.yml, I would not mind at all.

### CODE REVIEW

**Comment by webknjaz** [comment]
Taking over. I'll merge this into the main workflow and make sure that the bugs introduced are taken care of.

**Comment by ssbarnea** [comment]
> Taking over. I'll merge this into the main workflow and make sure that the bugs introduced are taken care of.

I am happy to see you taking this over. As long we endup with the same outcome (use of wsl pipeline) we should be fine.

While doing the surgery on ci.yml, please keep in mind that we may still need add a classic Windows pipeline in the future (non-wsl), one that tests that we are able to emit a meaningful error message when run in an unsupported environment, example https://github.com/ansible/ansible-language-server/pull/195

File: .github/workflows/npm.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          - test
+        upload-artifact:
+          - false
+        name:
+          - false
+        include:
+          - name: test (wsl)
+            npm-target: test
+            # https://github.com/actions/virtual-environments/issues/5151
+            os: windows-2019
```
Comment (ssbarnea) [inline .github/workflows/npm.yml:42]: That is needed as windows-2022 wsl seems unstable. Read https://github.com/actions/virtual-environments/issues/5151

**Comment by ssbarnea** [comment]
It should be noted that this was needed after more than 3 weeks of waiting. 

I would expect @webknjaz to remove this workflow file as part of his integrating with existing pipeline whant.

**Pull request merged**